FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/DataProcessor/DataConverter/FhirConverter.cs

```diff
@@ -0,0 +1,61 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Liquid.Converter;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Microsoft.Health.Fhir.Liquid.Converter.Processors;
+using Microsoft.Health.Fhir.Synapse.Common.Models.Data;
+using Microsoft.Health.Fhir.Synapse.Core.Exceptions;
+using Microsoft.Health.Fhir.Synapse.SchemaManagement.ContainerRegistry;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.DataProcessor.DataConverter
+{
+    public class FhirConverter
```

Comment:
Maybe we can name the data converter as `DefaultSchemaConverter` and `CustomSchemaConverter` ? Thus the data flow could be first convert to specific schema, and then write to parquet.